### PR TITLE
OSD: New pixel format, QUADSPI video driver

### PR DESCRIPTION
**Pull request by mluessi**
Needed for BrainFPV RE1. In contrast to Brain, which uses 2 buffers and 2 SPIs to render the OSD, RE1 uses a single buffer (2 bits/pixel) and the QUADSPI to output the OSD data. The horizontal timing is handled externally, which simplifies the STM32 code quite a bit. This required changes to the drawing routines and a new driver (`pios_video_quadspi.c`). It also needed a different encoding of fonts and images, so scripts to generate the font headers from PNGs have been added.

### CODE REVIEW

File: flight/Modules/OnScreenDisplay/inc/encode_fonts.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# The first 3 fonts are the small / medium / large fonts selectable by the user
+./encode_fonts.py -s 32 -l 127 -o ../fonts.c -d fonts.h font8x10.png font_outlined8x14.png font12x18.png font_outlined8x8.png
```
Comment (mlyle) [inline flight/Modules/OnScreenDisplay/inc/encode_fonts.sh]: newline at end of file

**Comment by mlyle** [comment]
With that one newline I think this is mergable

**Comment by mluessi** [comment]
@mlyle thanks! Newline added

**Comment by mlyle** [comment]
jenkins, build artifacts please

**Comment by mlyle** [comment]
(Just want to verify Mac & Windows builds)

**Pull request merged**